src/node/coinstats.cpp

```diff
@@ -111,6 +124,10 @@ bool GetUTXOStats(CCoinsView* view, CCoinsStats& stats, CoinStatsHashType hash_t
     case(CoinStatsHashType::NONE): {
         return GetUTXOStats(view, stats, nullptr, interruption_point);
     }
+    case(CoinStatsHashType::MUHASH): {
```

Comment:
e6856a3809e54dc4de03cb4cd4f9d78495cc0625 nit: handle `MUHASH` before `NONE`? (same with `PrepareHash ` and `ParseHashType`)

Reply:
Done.